thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -178,16 +224,34 @@ public void run() {
     executionContext.setExecutionResult(ALERT_CONFIG, alertConfig);
   }
 
-  protected String getFunctionTypeByTimeGranularity(TimeGranularity timeGranularity) {
+  protected AnomalyFunctionDTO getDefaultFunctionSpecByTimeGranularity(TimeGranularity timeGranularity) {
+    AnomalyFunctionDTO anomalyFunctionSpec = new AnomalyFunctionDTO();
     switch (timeGranularity.getUnit()) {
       case MINUTES:
```

Comment:
Are we casting all 1_MINUTES, 5_MINUTES, 15_MINUTES into "MINUTES" here?

Reply:
Yes. For 1, 5, and 15 minutes, we can still use SIGN TEST. Currently I don't want to over complicate the problem. We can add more condition if we need specify differently for certain time granularity.